healthcheck.sh

```diff
@@ -1,15 +1,20 @@
 #!/usr/bin/env bash
 
+#set -x  # Print commands
+set -e  # Exit on errors
+
 # nginx config is valid
-nginx -t || exit 1
+echo "Checking nginx configuration..."
+nginx -t
 
 # supervisor services are running
 if [[ -f /run/supervisord.pid ]]; then
-	supervisorctl status docker-gen | grep RUNNING >/dev/null || exit 1
-	supervisorctl status nginx | grep RUNNING >/dev/null || exit 1
-	supervisorctl status crond | grep RUNNING >/dev/null || exit 1
 
-	exit 0
+	echo "Checking supervisor services..."
+	# Since "supervisorctl status" is heavy and the healthcheck runs quit often, we use "ps" here
+	pslist=$(ps)
+	echo ${pslist} | grep docker-gen >/dev/null
+	echo ${pslist} | grep "nginx: master process" >/dev/null
+	echo ${pslist} | grep "nginx: worker process" >/dev/null
+	echo ${pslist} | grep crond >/dev/null
 fi
```

Comment:
```suggestion
else
	exit 1
fi
```

Reply:
Removed if altogether. Flat and simple now.